codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala

```diff
@@ -293,10 +293,12 @@ object ActionServiceSourceGenerator {
     val classNameAction = service.className
     val protoName = service.fqn.protoName
 
+    val relevantDescriptors =
+      collectRelevantTypes(service.commandTypes, service.fqn)
+        .collect { case fqn if fqn.isProtoMessage => s"${fqn.parent.javaOuterClassname}.getDescriptor()" }
+
     val descriptors =
-      (collectRelevantTypes(service.commandTypes, service.fqn)
-        .map(d =>
-          s"${d.parent.javaOuterClassname}.getDescriptor()") :+ s"${service.fqn.parent.javaOuterClassname}.getDescriptor()").distinct.sorted
+      (relevantDescriptors :+ s"${service.fqn.parent.javaOuterClassname}.getDescriptor()").distinct.sorted
```

Comment:
relevantDescriptors are used to generate the `additionalDescriptors` in the provide, but we don't want the POJOs in this list.

Reply:
Where would the POJO come from for an Action, it doesn't have state?